### PR TITLE
Remove last-rited qt4/qt3support-rdeps 

### PR DESCRIPTION
**Pull request by a17r**

### CODE REVIEW

**Comment by a17r** [comment]
@aballier Please ack texlive-core cleanup (ignore the rest). The above commits would only leave `2017-r3` and `2015-r1`, the latter blocks poppler cleanup until we have figured out the texlive-basic-2015 situation.

**Comment by a17r** [comment]
@aballier please ack the texlive-{core,basic} cleanup.

File: app-text/texlive-core/texlive-core-2015-r1.ebuild [hunk 1]
```diff
@@ -1,344 +0,0 @@
-# Copyright 1999-2018 Gentoo Foundation
-# Distributed under the terms of the GNU General Public License v2
-
```
Comment (aballier) [inline app-text/texlive-core/texlive-core-2015-r1.ebuild]:
```
alexis@pepito ~/dev/gentoo-x86 $ ls dev-texlive/*/*2015* | wc -l
37
```

all those need to go before removing -core

File: app-text/texlive-core/texlive-core-2016-r6.ebuild [hunk 2]
```diff
@@ -1,354 +0,0 @@
-# Copyright 1999-2018 Gentoo Foundation
-# Distributed under the terms of the GNU General Public License v2
-
```
Comment (aballier) [inline app-text/texlive-core/texlive-core-2016-r6.ebuild]:
```
alexis@pepito ~/dev/gentoo-x86 $ ls dev-texlive/*/*2016* | wc -l
36
```

ditto here

Comment (aballier) [reply app-text/texlive-core/texlive-core-2016-r6.ebuild]: plus 2016 is last arm64 stable, not sure how this even passes CI

**Comment by a17r** [comment]
@aballier updated. arm64 now has the choice of either stabilising texlive-core-2016-r6 or the texlive-2017 real deal.

File: profiles/package.mask [hunk 3]
```diff
@@ -29,6 +29,12 @@
 
 #--- END OF EXAMPLES ---
 
+# Andreas Sturmlechner <asturm@gentoo.org> (06 Apr 2018)
+# Obsolete dev-texlive/* cleanup. Masked for removal in 30 days.
+dev-texlive/texlive-documentation-base
+dev-texlive/texlive-mathextra
+dev-texlive/texlive-science
```
[the remark this answers is not in the document]
Comment (a17r) [reply profiles/package.mask]: yes, I thought you would need to last-rite them in any case. but I can drop the commit and leave it to you.

**Comment by a17r** [comment]
Closing for now as texlive-*-2016 cleanup is blocked by dev-tex/biblatex stabilisation.

**Pull request closed without merging**